Docs: Addition of more prominent link to docker conf (#46504)
grafana/grafana: docs/sources/installation/docker.md

Each edition is available in two variants: Alpine and Ubuntu. See below.

> For documentation regarding configuration of a docker image, please see [configure a Grafana Docker image](https://grafana.com/docs/grafana/latest/administration/configure-docker/).

Reviewer: ```suggestion
For documentation regarding the configuration of a docker image, refer to [configure a Grafana Docker image](https://grafana.com/docs/grafana/latest/administration/configure-docker/).
```
